Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.yml

```
@@ -3,16 +3,12 @@ commonfields:
   id: Palo Alto Networks Enterprise DLP
   version: -1
 configuration:
-- additionalinfo: Access token generated in the Enterprise DLP UI
-  display: Access Token
-  name: access_token
+- display: Access Token
+  displaypassword: Refresh Token
```

Because the new auth will use the Client ID & Client secret 
it's better to use those terms in the display
```suggestion
- display: Client ID
  displaypassword: Client Secret
```

**in addition** 
are the `client_id` and `client_secret` values the same as  the old values stored in `identifier` and `password`?
or user need to use another values ?

---

We are not replacing access token and refresh token with client id and client secrets. The support for client id and secret is in addition to access token/refresh tokens. So the user can choose to use either access token/refresh token or client id/client secret.

---

https://user-images.githubusercontent.com/96203646/172691709-1db4d734-e30f-455c-b98b-71152f08f28c.mov